File: backend/vertragsstand.py
"""
Die aktuellen Fassungen der Vertragstexte. Eine Quelle im Backend.

Die Registrierung protokolliert seit dem 10.09.2026, welche AGB- und
AVV-Fassung ein Kunde angenommen hat (vertragsannahmen). Bis dahin stand die
Fassung nur im Dashboard (register/page.tsx) und in der Landing
(lib/vertragsstand.ts). Das Backend konnte deshalb nicht sagen, ob eine
protokollierte Annahme noch die aktuelle Fassung betrifft, und genau das
braucht das Gate fuer die Bestandskonten: 16 Konten haben den AVV nie
angenommen, ohne ihn darf complyo fuer sie keine Besucherdaten verarbeiten.

Wer eine Fassung aendert, aendert sie hier UND in
dashboard-react/src/lib/vertragsstand.ts UND in
landing-react/src/lib/vertragsstand.ts. Der Waechter in
tests/test_vertragsstand.py vergleicht die drei.
"""

AGB_VERSION = "2026-09-01"
AVV_VERSION = "2026-09-10"

File: dashboard-react/src/lib/vertragsstand.ts
/**
 * Fassungen der Vertragstexte. Eine Quelle im Dashboard.
 *
 * Bis zum 11.09.2026 standen die Fassungen in register/page.tsx. Seitdem
 * braucht sie auch das Gate fuer Bestandskonten (components/auth/VertragsGate),
 * das die 16 Konten von vor dem 10.09.2026 um die AVV-Zustimmung bittet.
 * Zwei Kopien laufen auseinander, und eine protokollierte Annahme fuer eine
 * Fassung, die nie angezeigt wurde, ist kein Nachweis.
 *
 * Das Backend fuehrt dieselben Werte in backend/vertragsstand.py; der Test
 * backend/tests/test_vertragsstand.py vergleicht beide. Die Landing hat ihre
 * eigene Kopie in landing-react/src/lib/vertragsstand.ts (eigenes Paket).
 */
export const AGB_VERSION = '2026-09-01';
export const AVV_VERSION = '2026-09-10';
